examples/webgl_loader_amf.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>three.js webgl - AMF</title>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+		<style>
+			body {
+				font-family: Monospace;
+				background-color: #000000;
+				margin: 0px;
+				overflow: hidden;
+			}
+
+			#info {
+				color: #fff;
+				position: absolute;
+				top: 10px;
+				width: 100%;
+				text-align: center;
+				z-index: 100;
+				display:block;
+
+			}
+
+			a { color: skyblue }
+			.button { background:#999; color:#eee; padding:0.2em 0.5em; cursor:pointer }
+			.highlight { background:orange; color:#fff; }
+
+			span {
+				display: inline-block;
+				width: 60px;
+				float: left;
+				text-align: center;
+			}
+
+		</style>
+	</head>
+	<body>
+		<div id="info">
+			<a href="http://threejs.org" target="_blank">three.js</a>
+			<a href="http://amf.wikispaces.com" target="_blank">AMF File format</a>
+			<div>AMF loader test by <a href="https://github.com/tamarintech">tamarintech</a></div>
+			<div>Rook manufacturing file from <a href="http://amf.wikispaces.com/AMF+test+files">AMF test files</a></div>
+
+		</div>
+
+		<script src="../build/three.min.js"></script>
+		<script src="js/loaders/AMFLoader.js"></script>
+
+		<script src="js/Detector.js"></script>
+		<script src="js/controls/OrbitControls.js"></script>
+
+		<script>
+
+			if ( ! Detector.webgl ) Detector.addGetWebGLMessage();
+
+			var container;
+
+			var camera, controls, scene, renderer, grid;
+
+			var loader;
+
+			init();
+
+			function init() {
+
+				container = document.createElement( 'div' );
+				document.body.appendChild( container );
+
+				scene = new THREE.Scene();
+				scene.add( new THREE.AmbientLight( 0x999999 ) );
+
+				camera = new THREE.PerspectiveCamera( 35, window.innerWidth / window.innerHeight, 1, 25 );
+
+				scene.fog = new THREE.Fog( 0x999999, 0, 25 );
+
+				// Normally Z is up for objects intended to be 3D printed.
+				// This rook is sideways ;)
+
+				camera.up = new THREE.Vector3( 0, 1, 0 );
+				camera.position.set( 1, 6, 9);
+
+				camera.add(new THREE.DirectionalLight( 0xffffff, 0.8 ) );
+
+				scene.add(camera);
```

Comment:
Interesting.  I chose DirectionalLight because I thought it would more uniformly light the model and I believe it's cheaper than PointLight's with decay, shadows, etc.  As a relatively simple model created for reproduction (similar to STL) I think the cheaper and simpler lighting system is more than adequate.

Plus I haven't actually implemented loading normals from the AMF yet..... :8ball: 

22 of the 32 loader examples all use DirectionalLight (approximate by random grepping | wc).  We're going to have a lot to update when DirectionalLight gets a direction. :)